RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.15.0
```

Comment:
A change like that only merits a patch-level release! 3.14.4

Reply:
Maybe further changes will promote this 3.14.4 to 3.15.0... but let's stick to 3.14.4 for now

Reply:
Yes, but i saw this milestone https://github.com/VirtusLab/git-machete/milestone/29 and decided to include this issue as well, wdyt?

Reply:
Okay, let it be 3.15.0 then

Reply:
+1, makes sense!

Reply:
Or do people use another,better logic flow when considering releases?

Reply:
Generally I keep the version bump as low as possible (i.e. only bumping minor version once there's a backward-compatible change to the _protocol_ as defined in https://github.com/VirtusLab/git-machete/blob/master/CONTRIBUTING.md#versioning)

Reply:
But I wouldn't care too much, it's much more important in case of versioning libraries/APIs than end-user tools like git-machete

Reply:
Ok!